PWGMM/Mult/Tasks/dndeta-hi.cxx

```diff
@@ -56,8 +56,10 @@ using namespace o2::framework::expressions;
 using namespace o2::aod::track;
 
 using namespace o2::aod;
-using namespace o2::aod::evsel;
 using namespace o2::analysis;
+using namespace o2::aod::hf_cand_2prong;
+using namespace o2::aod::hf_cand_bplus;
+using namespace o2::analysis::hf_cuts_bplus_to_d0_pi;
```

Comment:
I do not see where the HF stuff is used in the task, why do you need these namespaces?